provider/resource_keycloak_ldap_user_federation_test.go

```diff
@@ -550,9 +710,15 @@ resource "keycloak_ldap_user_federation" "openldap" {
 	full_sync_period         = %d
 	changed_sync_period      = %d
 
+	server_principal                         = %s
+	use_kerberos_for_password_authentication = %t
+	allow_kerberos_authentication            = %t
+	key_tab                                  = %s
+	kerberos_realm                           = %s
```

Comment:
I think the issue with the failing tests is that all of the string parameters aren't surrounded in quotes.  Take a look at line 706 for an example.

Reply:
You are obviously right, sir! :)